script/test/test_benchmark.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# This script should return a non-zero value if either
+# linter fails or the pytest fails. This is important for the CI system.
+
+# temporarily remove __init__.py from root if it exists
+if [ -f ./__init__.py ]; then
+   mv ./__init__.py ./__init__.py.bak
+fi
+
+# Run black, isort, linter 
+sh script/formatting/pre-push.sh
+return_code=$?
+if [ $return_code -ne 0 ];
+then
+    exit $return_code
+fi
+
+# Run only benchmark tests
+PYTHONPATH=./ pytest test/ --cov-report term --cov-config=.coveragerc --cov=eva/ -s -v --log-level=WARNING ${1:-} -m "benchmark"
```

Comment:
Is it fine if we only run the benchmark command here? We can avoid doing linting etc

Reply:
I think its better for the linting check to be there, in case we need to run these tests separately. In that case better to have linting in there